views/homepage/error.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Wrong Username or Password</h1>
```

Comment:
We can replace this with `flash_messages` instead. I saw you included `sinatra-flash`. You can use that to implement flash messages that display when a user entered a wrong username/password or to display a success message when user has logged in.